CHANGELOG.md

```diff
@@ -6,6 +6,15 @@
 
 - Updated to GraphDB [11.1.0](https://graphdb.ontotext.com/documentation/11.1/release-notes.html#graphdb-11-1-0)
 - Updated the `extraObjects` template processing to additionally support rendering string Helm templates.
+- Added support for string template rendering in the following properties:
```

Comment:
Rebase the branch on develop/12.2 and update the release version in the changelog